easybuild/easyblocks/p/pytorch.py

```diff
@@ -490,14 +490,20 @@ def add_enable_option(name, enabled):
             # determine the compute capability of a GPU in the system and use that which may fail tests if
             # it is to new for the used nvcc
             env.setvar('TORCH_CUDA_ARCH_LIST', ';'.join(cuda_cc))
+            with_gpu_support = True
         else:
             # Disable CUDA
             options.append('USE_CUDA=0')
+            with_gpu_support = False
 
         if pytorch_version >= '2.0':
-            add_enable_option('USE_ROCM', get_software_root('ROCm'))
-        elif pytorch_version >= 'v1.10.0':
-            add_enable_option('USE_MAGMA', get_software_root('magma'))
+            has_rocm = get_software_root('ROCm')
+            add_enable_option('USE_ROCM', has_rocm)
+            if has_rocm:
+                with_gpu_support = True
+
+        if pytorch_version >= 'v1.10.0':
```

Comment:
is the `v` in the version intentional? I would have thought that might trip up the version comparison, even with `LooseVersion`?